src/components/backend-ai-agent-list.ts

```diff
@@ -613,8 +614,16 @@ export default class BackendAIAgentList extends BackendAIPage {
 
         <vaadin-grid-column resizable header="${_t("agent.Resources")}" .renderer="${this._boundResourceRenderer}">
         </vaadin-grid-column>
+        <vaadin-grid-sort-column width="100px" resizable path="scaling_group" header="${_t("general.ResourceGroup")}"></vaadin-grid-sort-column>
+
+
+
         <vaadin-grid-column width="130px" flex-grow="0" resizable header="${_t("agent.Status")}" .renderer="${this._boundStatusRenderer}"></vaadin-grid-column>
+
         <vaadin-grid-column resizable header="${_t("general.Control")}" .renderer="${this._boundControlRenderer}"></vaadin-grid-column>
+
+
+
```

Comment:
I think the cod looks good. But, is there any reason to add many blank lines here?

Reply:
Oh, that is my mistake. I'll delete blanks and push. Thank you for comment.